src/mudlet.cpp

```diff
@@ -565,6 +565,18 @@ mudlet::mudlet()
     connect(dactionIRC, &QAction::triggered, this, &mudlet::slot_irc);
     connect(dactionDiscord, &QAction::triggered, this, &mudlet::slot_discord);
     connect(dactionLiveHelpChat, &QAction::triggered, this, &mudlet::slot_irc);
+    connect(dactionShowErrors, &QAction::triggered, [=]() {
+        auto host = getActiveHost();
+        if (!host) {
+            return;
+        }
+        host->mpEditorDialog->slot_show_current();
+        host->mpEditorDialog->raise();
+        host->mpEditorDialog->showNormal();
```

Comment:
:thinking: If a user has the editor window opened full screen/maximised on a second (or higher) monitor than this will undo either of those choices - whilst that might be the case for only a very small set of users doing this might not be the optimum design...

Reply:
Instead we just need to clear the `Qt::WindowMinimized` flag if it is set - to de-iconify the window to whichever of normal, maximised or full-screen was previous set (and not try to set the `Qt::WindowActive` as that is not something that we should do according to the Qt Documentation):
```suggestion
        host->mpEditorDialog->setWindowState(host->mpEditorDialog->windowState() &~(Qt::WindowMinimized|Qt::WindowActive));
```

Humm, that *almost* works - except that the editor window does not get shown when this is called upon to *create* it in the first place.

Reply:
That doesn't sound like an improvement, then!